hack/create-identity-secret.sh

```diff
@@ -20,7 +20,8 @@ set +o xtrace
 
 # Install kubectl
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
-KUBECTL="${REPO_ROOT}/hack/tools/bin/kubectl"
+source "${REPO_ROOT::-3}/common-vars.sh"
```

Comment:
```suggestion
source "${REPO_ROOT}/hack/common-vars.sh"
```
I am more inclined to use the above suggestion over the current implementation because the above suggestion improves readability.
User on a first glance can figure out that `REPO_ROOT` stores the value of repository's root, and `common_vars.sh` lives in `hack/` dir from repo's root.

Reply:
Refer to line https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2523/files#diff-a281296dd4955d10bd904f8bf1d2c0ad0dcd36066f931b572aa606b4d85ea849R28 as an example.